.travis.yml

```diff
@@ -50,7 +50,7 @@ jobs:
         - if [ "$TRAVIS_SECURE_ENV_VARS" != "false" ]; then eval "$(ssh-agent -s)"; fi
         - if [ "$TRAVIS_SECURE_ENV_VARS" != "false" ]; then chmod 600 /tmp/id_rsa_downcloud_cozy-store; fi
         - if [ "$TRAVIS_SECURE_ENV_VARS" != "false" ]; then ssh-add /tmp/id_rsa_downcloud_cozy-store; fi
-        - curl -fsSL https://bootstrap.pypa.io/get-pip.py | python - --user
+        - curl -fsSL https://bootstrap.pypa.io/2.7/get-pip.py | python - --user
```

Comment:
Why do we need pip, and why do we need at 2.7 since there is no support for Python 2.7 anymore ?

Reply:
I think we do not need pip, it must be a remnant of the need for transifex. Can you try to remove it ?

Reply:
done in the next commit ;)